inlang/documentation/ecosystem/overview/introduction.md

```diff
@@ -1,6 +1,59 @@
-# Under construction
+# Introduction
 
-See old docs start page [here](/documentation/sdk)
+Welcome to the inlang documentation!
+
+---
+
+## What is inlang?
+
+![ecosystem illustration](https://cdn.jsdelivr.net/gh/opral/monorepo@latest/inlang/documentation/ecosystem/assets/ecosystem-atom.jpg)
+
+<br>
+
+inlang offers tailored localization products for each [use case](/#personas) (Developer, Designer, Translator). You only have to set inlang up once; then, you can use every inlang app out of the box. 
+
+Some of the main inlang features include:
+
+- **Decentralized workflows** - Don't force devs in a cloud solution; let devs, translator and designer work in their native environment. ([Apps](/c/apps))
+- **One source of truth** - Never have syncing problems again; The data is always stored in your repo while collaborator can work on it in their own interface.
```

Comment:
how about
> while collaborators work with their own interface -- no git knowledge required.

Reply:
@jldec Don't mention git. Nobody outside of software eng knows what git is. Use "change control" instead.

I would change the one source of truth to:

```diff
-**One source of truth** - Never have syncing problems again; 
- The data is always stored in your repo while collaborator can work on it in their own interface.
+ **Uses repositories** - Data is stored in lix repositories to provide every inlang project with change control.
```